### DISCUSSION TITLE
More keybind possibilities for Taiko, please!

### DISCUSSION THREAD

**Discussion by DaVince** [ppy/osu#29737] [Ideas] [answered]
osu!taiko allows me to remap the input to my taiko controller, so of course I did that. But sometimes I'm in a situation where I'd like to play with the keyboard, and then I find myself in a pickle.

![Screenshot_20240906_204438](https://github.com/user-attachments/assets/9e5ff2b4-5134-40ab-b40a-f4cb1d012108)

As shown, it's possible to bind two inputs to the left rim and centre. But for whatever reason, I can only bind _one_ input to the right rim and centre. I'd love it if it allowed me to bind some keys there, too.

Heck, I'd love it if the input system was reevaluated entirely, allowing for arbitrarily many keybinds on _any_ action, but that'd be asking for much more than just this small fix, haha.

**Comment by Joehuu** [comment, marked as answer]
This is a duplicate of https://github.com/ppy/osu/discussions/28836. Please upvote that!
